src/dmx/pixel.cpp

```diff
@@ -10,12 +10,12 @@ namespace dmxfish::dmx {
     [[nodiscard]] std::string pixel::str() {
 		std::stringstream ss;
         if (this->iluminance >= 0.) {
```

Comment:
do we need still need the condition?
getHue etc. should convert it automatically.
and the output has side effects, should it not output the values directly, so we can better investigate other problems? Maybe with a comment, which parts is invalid (or valid)?